.github/workflows/ci.yml

```
@@ -25,13 +25,58 @@ jobs:
     with:
       SOLUTION_FILE_PATH: 'source/GreenEnergyHub.Charges/GreenEnergyHub.Charges.sln'
+
+  # Run tests in 'IntegrationTests.dll' with category ChargeIngestionTestsChargeInformation'
+  dotnet_solution_ci_test_integrationtests_part1:
+    uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci-test.yml@v8
+    with:
+      TESTS_DLL_FILE_PATH: '\tests\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.IntegrationTests\bin\Release\net6.0\GreenEnergyHub.Charges.IntegrationTests.dll'
+      TESTS_FILTER_EXPRESSION: 'Category=ChargeIngestionTestsChargeInformation'
+      ASPNETCORE_TEST_CONTENTROOT_VARIABLE_NAME: ASPNETCORE_TEST_CONTENTROOT_GREENENERGYHUB_CHARGES_WEBAPI
+      ASPNETCORE_TEST_CONTENTROOT_VARIABLE_VALUE: '\tests\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.WebApi'
+      DOWNLOAD_ATTEMPT_LIMIT: 12
+      USE_AZURE_FUNCTIONS_TOOLS: true
+      CODE_COVERAGE_FLAGS: business
+    secrets:
+      AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      AZURE_SPN_ID: ${{ secrets.AZURE_SPN_ID_OIDC }}
+      AZURE_KEYVAULT_URL: ${{ secrets.AZURE_KEYVAULT_URL }}
+      AZURE_SECRETS_KEYVAULT_URL: ${{ secrets.AZURE_SECRETS_KEYVAULT_URL }}
+
+  # Run tests in 'IntegrationTests.dll' with category ChargeIngestionTestsChargePrices'
+  dotnet_solution_ci_test_integrationtests_part2:
+    uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci-test.yml@v8
+    with:
+      TESTS_DLL_FILE_PATH: '\tests\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.IntegrationTests\bin\Release\net6.0\GreenEnergyHub.Charges.IntegrationTests.dll'
+      TESTS_FILTER_EXPRESSION: 'Category=ChargeIngestionTestsChargePrices'
+      ASPNETCORE_TEST_CONTENTROOT_VARIABLE_NAME: ASPNETCORE_TEST_CONTENTROOT_GREENENERGYHUB_CHARGES_WEBAPI
+      ASPNETCORE_TEST_CONTENTROOT_VARIABLE_VALUE: '\tests\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.WebApi'
+      DOWNLOAD_ATTEMPT_LIMIT: 12
+      USE_AZURE_FUNCTIONS_TOOLS: true
+      CODE_COVERAGE_FLAGS: business
+    secrets:
+      AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      AZURE_SPN_ID: ${{ secrets.AZURE_SPN_ID_OIDC }}
+      AZURE_KEYVAULT_URL: ${{ secrets.AZURE_KEYVAULT_URL }}
+      AZURE_SECRETS_KEYVAULT_URL: ${{ secrets.AZURE_SECRETS_KEYVAULT_URL }}
+
+  # Run tests in 'IntegrationTests.dll' NOT covered by previous cases
+  dotnet_solution_ci_test_integrationtests_part3:
+    uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci-test.yml@8.7.0
+    with:
+      TESTS_DLL_FILE_PATH: '\tests\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.IntegrationTests\bin\Release\net6.0\GreenEnergyHub.Charges.IntegrationTests.dll'
+      TESTS_FILTER_EXPRESSION: 'Category!=ChargeIngestionTestsChargeInformation!=ChargeIngestionTestsChargePrices&Category'
```

Filter value seems a little bit strange to me.

Shouldn't it be:

```suggestion
      TESTS_FILTER_EXPRESSION: 'Category!=ChargeIngestionTestsChargeInformation&Category!=ChargeIngestionTestsChargePrices'
```